examples/computer_vision_techniques/loop_edge_enhance.py

```diff
@@ -21,7 +21,7 @@
 aia = sunpy.map.Map(AIA_171_IMAGE)
 bottom_left = SkyCoord(750 * u.arcsec, -200 * u.arcsec, frame=aia.coordinate_frame)
 top_right = SkyCoord(1500 * u.arcsec, 550 * u.arcsec, frame=aia.coordinate_frame)
-aia_smap = aia.submap(bottom_left, top_right=top_right)
+aia_smap = aia.submap(bottom_left, top_right)
```

Comment:
I've done this to make sure the warning is treated as an error.
This should be undone.